Extension/src/LanguageServer/configurations.ts

```diff
@@ -16,7 +16,7 @@ import { getCustomConfigProviders } from './customProviders';
 import { SettingsPanel } from './settingsPanel';
 import * as os from 'os';
 import escapeStringRegExp = require('escape-string-regexp');
-import * as jsonc from 'jsonc-parser';
+import * as jsonc from 'comment-json';
```

Comment:
This file has to references to JSON.stringify that should be changed to jsonc.stringify.